### PR TITLE
Feature : internationalization part 1 - Feedback, Metadata curator, Ontology new, Feedback and Home pages  

### PR DESCRIPTION
**Pull request by SirineMhedhbi**

### CODE REVIEW

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
Seems there is an issue in the yaml file and don't pass the tests 
image

File: config/locales/en.yml [hunk 1]
```diff
@@ -429,4 +460,285 @@ en:
...
+      please upload a full version of your ontology.
+    submitting_ontology_view: Thank you for submitting your ontology view to %{site}.
+    submitting_ontology: Thank you for submitting your ontology to %{site}.
+    processing_message: |
+      We will now put your ontology in the queue to be processed. Please keep in mind that it may take up to several
+      hours before %{site} users will be able to explore and search your ontology.
+    notification_message: "When your ontology is ready for viewing, you will receive an email notification and it will be available here:"
+    contact_support: "If you have any questions or problems, please email the %{site} support team at:"
+    collecting_data_message: We are still collecting data for %{acronym}
+    submit_new_ontology: Submit New Ontology
```
Comment (syphax-bouazzouni) [inline config/locales/en.yml:615]: this need to be `Submit new ontology` not `Submit New Ontology` to pass the system tests

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/views/home/feedback/feedback.html.haml [hunk 2]
```diff
@@ -14,45 +14,45 @@
...
         %p.feedback-input-title
-          Email
+          = t("home.feedback.email")
           %font{:color => "red"} *
         %input.feedback-input{:name => "email", :type => "email"}/
         %p.feedback-input-title
-          Name
-          %font.register-optional (optional)
+          = t("home.feedback.name")
+          %font.register-optional= t("home.feedback.optional")
```
Comment (Bilelkihal) [inline app/views/home/feedback/feedback.html.haml:35]: For '(optional)' it's better to keep the parentheses in the UI, so we keep the local lang files clean with only texts,
Like this for example:
`"(#{t('home.feedback.optional')})"`

File: app/views/home/feedback/feedback.html.haml [hunk 3]
```diff
@@ -14,45 +14,45 @@
...
         %input.feedback-input{:name => "name", :type => "text"}/
       - else
         = hidden_field_tag "name", session[:user].username
         = hidden_field_tag "email", session[:user].email
       %p.feedback-input-title
-        Error/Proposition page URL
-        %font.register-optional (optional)
+        = t("home.feedback.proposition_url_page")
+        %font.register-optional= t("home.feedback.optional")
```
Comment (Bilelkihal) [inline app/views/home/feedback/feedback.html.haml:42]: Same as above

File: app/views/home/feedback/feedback.html.haml [hunk 4]
```diff
@@ -14,45 +14,45 @@
...
       %p.feedback-input-title
-        Feedback
+        = t("home.feedback.feedback")
         %font{:color => "red"} *
       %textarea.feedback-input{:name => "comment", :rows => "6"}
       %p.feedback-input-title
-        Tags
-        %font.register-optional (optional)
+        = t("home.feedback.tags")
+        %font.register-optional= t("home.feedback.optional")
```
Comment (Bilelkihal) [inline app/views/home/feedback/feedback.html.haml:50]: Same as above

File: app/views/home/index.html.haml [hunk 5]
```diff
@@ -75,7 +75,7 @@
         .home-services-buttons.container
           .insert-sample-text-button{'data-action': 'click->sample-text#annotator_recommender', 'data-sample-text': t("sample_text")}
             %div
-              Insert sample text
+              = t('home.insert_sample_text')
```
Comment (Bilelkihal) [inline app/views/home/index.html.haml:78]: insert_sample_text already exists in annotator and recommender, 
replace this with:
= t('insert_sample_text')

File: app/views/home/index.html.haml [hunk 6]
```diff
@@ -134,43 +134,43 @@
...
+            %p= t("home.mappings")
         .home-statistics-item
           %hr/
           %div
             %h4
               = format_number_abbreviated(@users_count)
-            %p Users
+            %p= t("home.users")
       .home-statistics-link.justify-content-end
         = link_to 'See details','/statistics', target: "_blank"
```
Comment (Bilelkihal) [inline app/views/home/index.html.haml:175]: You forgot this 'See details' text

**Review by Bilelkihal — CHANGES_REQUESTED** [review]

**Review by Bilelkihal — COMMENTED** [review]
Great work

**Pull request merged**